README.md

```diff
@@ -81,11 +185,85 @@ pnpm test:watch
 
 ## Project Structure
 
-- `src/credentials/` - Contains credential definitions
-- `src/utils/` - Contains utility classes for API interaction
-- `src/nodes/` - Contains the node implementations
+- `src/credentials/` - Contains credential definitions for Pulse API authentication
+- `src/nodes/` - Contains all node implementations:
+  - `AccountAction/` - Account management nodes
+  - `OfficeAction/` - Office, employee, and planning management nodes
+  - `OrganizationsAction/` - Organization and directory management nodes
+  - `PeopleAction/` - People record management nodes
+  - `RecruitmentAction/` - Candidate and quiz management nodes
+  - `TalentAction/` - Talent profile management nodes
+  - `WorkflowAction/` - Project and workflow management nodes
+  - `common/` - Common base classes and utilities for nodes
+- `src/utils/api/` - Contains utility classes for API interaction with Pulse platform
 - `src/__tests__/` - Contains tests for all components
-- `index.ts` - Exports the credentials and nodes
+- `index.ts` - Main entry point that exports the credentials and nodes
+
+## Usage Examples
+
+### Example 1: Creating a Talent Profile with Resume Upload and Add New Experience entry
```

Comment:
Examples should be related to customer cases. Internal usage cases can be explained in internal documentation. Our customers won't get access to those nodes due to API scope limitation on client token. Can you change example to show some related to team membership, project documents, work unit creation?